R/trace_tests.R

```diff
@@ -145,7 +145,7 @@ new_test_counter <- function(key) {
   .counters[[key]]$tests$.data <- vector("integer", 3L)
   .counters[[key]]$tests$.value <- integer(1L)
   .counters[[key]]$tests$tally <- matrix(
-    0L,
+    NA_integer_,
```

Comment:
An minor, unrelated change. Using `NA` as a default value instead of `0`. Functionally, this has no impact in almost all cases. I've only seen one example where this value might have any effect, which is when testing interrupts. I found one instance where a counter's value was incremented, but the test hit wasn't recorded before the interrupt took effect.